code/game/machinery/doors/airlock.dm

```diff
@@ -1070,3 +1070,9 @@ About the new airlock wires panel:
 		src.open()
 		src.lock()
 	return
+
+/obj/machinery/door/airlock/proc/is_emagged() // used to keep people going through taped closed doors

```

Comment:
Just tested with the emagged var, it doesn't work. I guess that's why they used other methods in airlock.dm.